runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerPlugin.java

```diff
@@ -69,8 +69,9 @@
 	private static final String CONFIG_GROUP = "groundMarker";
 	private static final String MARK = "Mark tile";
 	private static final String WALK_HERE = "Walk here";
+	private static final String REGION_PREFIX = "region_";
 
-	private static final Gson gson = new Gson();
+	private static final Gson GSON = new Gson();
```

Comment:
Maybe use RuneLiteAPI.GSON (I think its even injectable via Guice, we should maybe start migrating to that)

Reply:
Based on Discord discussion, I'll pass on doing that for this PR. We might look at making a plugin-specific GSON instance later on for use among plugins for config serialization.